packages/app/src/contacts/components/contacts/delete-contacts-popup/delete-contacts-popup.component.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { useSelector } from "react-redux"
+import { createSelector } from "@reduxjs/toolkit"
+import { contactsStateSelector } from "App/contacts/selectors"
+import InfoPopup from "App/ui/components/info-popup/info-popup.component"
+import { defineMessages } from "react-intl"
+import { textFormatters } from "App/__deprecated__/renderer/utils/intl"
+import React, { useEffect, useState } from "react"
+import { FunctionComponent } from "App/__deprecated__/renderer/types/function-component.interface"
+
+export const messages = defineMessages({
+  deletedInfo: { id: "module.contacts.deletePopup" },
+})
+
+const DeleteContactsPopup: FunctionComponent = () => {
+  const getContactsCount = createSelector(
```

Comment:
What do you think to move this selector to "App/contacts/selectors"? 🤔 it is quite likely that the selector could be used again from another place.

Reply:
🟡 I think we shouldn't declare selectors inside of components because it will be recreated with each rerender